.gitignore

```diff
@@ -20,7 +20,10 @@ coverage
 .lock-wscript
 
 # Ignore all build output
-build/
+es5
+es2015
+es2018
+index.d.ts
```

Comment:
I'm tempted to suggest pointing to the es5 typings, just so that we don't have a generated file in the root of the repo, but it's not a big deal and we have already use the same pattern in other repos anyway.